### PR TITLE
TFlite layer tests

### PR DESCRIPTION
**Pull request by rnugmanx**
Ticket: https://jira.devtools.intel.com/browse/CVS-99807

### CODE REVIEW

**Review by jane-intel — APPROVED** [review]

File: tests/layer_tests/common/tf_layer_test_class.py [hunk 1]
```diff
@@ -61,18 +89,48 @@ def get_framework_results(self, inputs_dict, model_path):
...
                 tf.compat.v1.import_graph_def(graph_def, name='')
 
-                input = dict()
-                for key in inputs_dict.keys():
-                    data = inputs_dict.get(key)
-                    if self.use_old_api or self.use_new_frontend:
-                        key += ':0'
-                    input[key] = transpose_nchw_to_nhwc(data, self.use_new_frontend, self.use_old_api)
-
-                tf_res = sess.run([out + ":0" for out in outputs_list], input)
```
Comment (rkazants) [inline tests/layer_tests/common/tf_layer_test_class.py:71]: why did you remove it? Will it affect TF layer tests?

File: tests/layer_tests/common/tf_layer_test_class.py [hunk 2]
```diff
@@ -1,6 +1,5 @@
 # Copyright (C) 2018-2022 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
-
```
Comment (rkazants) [inline tests/layer_tests/common/tf_layer_test_class.py:3]: Please add blank line between code and the copyright

File: tests/layer_tests/conftest.py [hunk 3]
```diff
@@ -70,6 +70,11 @@ def pytest_addoption(parser):
         action="store_true",
         help="Use old API for model processing in Inference Engine",
     )
+    parser.addoption(
+        "--tflite",
+        required=False,
+        action="store_true",
+        help="Switch to tflite tests version")
```
Comment (rkazants) [inline tests/layer_tests/conftest.py:78]: Why are we sure that now TF layer tests will be executed in regular mode?

**Review by rkazants — CHANGES_REQUESTED** [review]

**Comment by rkazants** [comment]
@rnugmanx, please do not merge. I need to double-check that it will not affect testing for TF FE.
Thanks!

**Review by rkazants — COMMENTED** [review]
How can we skip some layer test for TFLite, if there is a bug in TFLite and no bug in TF FE or vice versa?

File: tests/layer_tests/common/tf_layer_test_class.py (hunk 1 above)
Comment (rnugmanx) [reply tests/layer_tests/common/tf_layer_test_class.py:71]: I didn't remove it, please check https://github.com/openvinotoolkit/openvino/blob/8e0e6983124a0238418dda6cd430623132ce5222/tests/layer_tests/common/tf_layer_test_class.py#L92

**Comment by rnugmanx** [comment]
> How can we skip some layer test for TFLite, if there is a bug in TFLite and no bug in TF FE or vice versa?

There will be separate run with --tflite key, and for each run its possible in pytest to exclude some tests, for example: "not Add"

**Comment by rkazants** [comment]
> > How can we skip some layer test for TFLite, if there is a bug in TFLite and no bug in TF FE or vice versa?
> 
> There will be separate run with --tflite key, and for each run its possible in pytest to exclude some tests, for example: "not Add"

How it will be convenient for us, if the config file is in the private directory and I want to include some test (for new operation) into the pre-commit?

**Comment by rnugmanx** [comment]
> @rnugmanx, please do not merge. I need to double-check that it will not affect testing for TF FE. Thanks!
TF FE and TFLite will be separate runs

**Comment by rnugmanx** [comment]
> > > How can we skip some layer test for TFLite, if there is a bug in TFLite and no bug in TF FE or vice versa?
> > 
> > 
> > There will be separate run with --tflite key, and for each run its possible in pytest to exclude some tests, for example: "not Add"
> 
> How it will be convenient for us, if the config file is in the private directory and I want to include some test (for new operation) into the pre-commit?

What is config file? If you mean config file for pre-commit tests and you need to add some tests only for pre-commit just add them - TFLite tests will have their own configuration for run

File: tests/layer_tests/common/tf_layer_test_class.py [hunk 4]
```diff
@@ -1,6 +1,5 @@
...
 import os
```
Comment (jane-intel) [inline tests/layer_tests/common/tf_layer_test_class.py:3]:
```suggestion

import os
```

**Review by jane-intel — APPROVED** [review]

**Review by rkazants — APPROVED** [review]
